kotlin/viewshed-location/README.md

```diff
@@ -0,0 +1,34 @@
+# Viewshed location
+
+Perform a viewshed analysis from a defined vantage point.
+
+![Image of viewshed location](viewshed-location.png)
+
+## Use case
+
+A 3D viewshed analysis is a type of visual analysis you can perform on a scene. The viewshed shows what can be seen from a given location. The output is an overlay with two different colors - one representing the visible areas (green) and the other representing the obstructed areas (red). Viewshed analysis is a form of "exploratory analysis", which means the results are calculated on the current scale of the data, and the results are generated very quickly. If more "conclusive" results are required, consider using a `GeoprocessingTask` to perform a viewshed instead.
+
+## How to use the sample
+
+Use the sliders to change the properties (heading, pitch, etc.), of the viewshed and see them updated in real time. To move the viewshed, double touch and drag your finger across the screen. Lift your finger to stop moving the viewshed.
```

Comment:
looking at the common design png (and after comparing with the Java SE sample), it looks as though we're missing the option to turn the visibility of both the viewshed and the frustum on and off. Apologies if I've missed a reason why that wasn't included: I know that our mobile apps are more restricted for UI than desktop (e.g. it looks like Java SE went one step further than the design and added in additional controls to change the colour of the viewshed!)

Reply:
just noticed it was missing from the original Android java sample too: I think for parity with other SDKs, we should include those check boxes for visibility and frustum if it's not too overwhelming on the UI. e.g. from Java SE:

```
visibilityToggle.selectedProperty().addListener(e -> viewshed.setVisible(visibilityToggle.isSelected()));
```

```
frustumToggle.selectedProperty().addListener(e -> viewshed.setFrustumOutlineVisible(frustumToggle.isSelected()))
```